### PR TITLE
Italian Translations

### PR DESCRIPTION
**Pull request by nicolo-ribaudo**
- Italian translations
- Now we can translate the title and the _Check Out_ button of the _Popular Extensions_ section

### CODE REVIEW

File: locales/it/translation.json [hunk 1]
```diff
@@ -0,0 +1,183 @@
+{
+    "_last.translated": "e1598a31476b10bbfdd1c83614307b2e92bb949d",
+    "lang": "it",
+    "index": {
+        "head": {
+            "title": "Brackets -  Un editor di testo open source e moderno, che capisce il web design."
```
Comment (marcelgerber) [inline locales/it/translation.json]: Is the doubled space (after `-`) intended?

Comment (nicolo-ribaudo) [inline locales/it/translation.json]: I'm sorry, I've corrected it.

**Comment by nicolo-ribaudo** [comment]
I received this email from GitHub.
Is it a problem?

**Comment by marcelgerber** [comment]
@nicolo-ribaudo Nope, everything's fine.

**Comment by nicolo-ribaudo** [comment]
@ingorichter, what about merging?

**Comment by marcelgerber** [comment]
@nicolo-ribaudo Sorry for having this hang around this long!
I'd like to start review this week.

**Comment by marcelgerber** [comment]
@Denisov21 Would you mind taking a look at this translation?
I've also set up a live demo here: https://dc7296e2001a9014f1d00ea89b966216336fe5b7.googledrive.com/host/0B986Uw1nLGY7XzNlY2hXVXI5X1U/index.html?lang=it, so you can see the strings used in context.

Note: Please look at both the homepage and the Contribute (Contribuisci) page.

**Comment by ingorichter** [comment]
Thanks for taking care of this PR @MarcelGerber.

**Comment by marcelgerber** [comment]
Thanks for the update. I've just reviewed the site (I've updated my above demo with your latest translation) using Google Translate, and content-wise it looks good.

I'd still like to have one native speaker review it, so @Denisov21 @Fredev would you be interested? (See https://github.com/adobe/brackets.io/pull/130#issuecomment-101335127)

**Comment by Denisov21** [comment]
Ok, @MarcelGerber! I will check!

**Comment by marcelgerber** [comment]
Thanks! Just go to the demo site referenced above (please review both the homepage and the [Contribute (Contribuisci)](https://dc7296e2001a9014f1d00ea89b966216336fe5b7.googledrive.com/host/0B986Uw1nLGY7XzNlY2hXVXI5X1U/contribute.html?lang=it) page) and look for phrases that could be improved and grammatical mistakes.

**Comment by Denisov21** [comment]
Yes I am interesting @MarcelGerber .

**Comment by Denisov21** [comment]
@MarcelGerber I find two small errors: How do I correct them?

**Comment by Denisov21** [comment]
I find in Italian language these errors (see image)!
immagine 1 14052015

File: locales/it/translation.json [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+                    },
+                    "live-preview": {
+                        "header": "Anteprima Live",
+                        "content": "Stabilisci una connessione in tempo reale con il tuo browser. Modifica il codice CSS e HTML e vedrai subito questi cambiamenti sullo schermo. Guarda anche dove sono applicati i selettori CSS nel browser semplicemente posizionandovi sopra il cursore. Questa è la potenza di un editor con gli strumenti di sviluppo integrati nel browser."
+                    },
+                    "preprocessor": {
+                        "header": "Supporto ai Preprocessori",
+                        "content": "Lavorare con i preprocessori in un modo completamente nuovo. Sappiamo quanto questi siano importanti nel tuo lavoro. Ecco perché vogliamo rendere Brackets l'editor con il miglior supporto ai preprocessori. Con brackets puoi utilizzare l'Anteprima Live o modificare velocemente i tuoi file LESS e SCSS per rendere il lavoro più facile che mai."
```
Comment (marcelgerber) [inline locales/it/translation.json]: Review comment per @Denisov21:
Always capitalize `Brackets`: `Con brackets` -> `Con Brackets`

File: locales/it/translation.json [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+                    "content": "L'estensione Extract per Brackets (Anteprima) to permette di ricavare gli stili di un PSD: colori, fonts, sfumature e altre informazioni, per scrivere del CSS pulito e minimale tramite i suggermenti del codice. Puoi anche estrarre livelli come immagini, usare le informazioni del PSD per definire variabili per i processori e misurare facilmente la distanza tra gli oggetti. Tutto con il codice sempre sotto gli occhi.",
+                    "more": "Altro …",
+                    "download": "Scarica Brackets + Extract (Anteprima) by Adobe"
+                },
+                "feature-highlights": {
+                    "inline-editor": {
+                        "header": "Editor Inline",
+                        "content": "Anziché spostarsi da un file all'altro, Brackets permette di aprire una finestra nel codice che ti interessa. Vuoi lavorare sul CSS applicato ad uno specifico ID? Posiziona il cursore su quell'ID, premi <kbd>Command</kbd> / <kbd>Ctrl+E</kbd> e Brackets mostrerà tutti i selettori CSS con quell'ID in una finestra nel il codice, così che tu possa lavorare facilmente su diversi file del tuo progetto."
```
Comment (marcelgerber) [inline locales/it/translation.json]: Review comment per @Denisov21:
Drop the `nel`: `in una finestra nel il codice` -> `in una finestra il codice`

**Comment by marcelgerber** [comment]
@Denisov21 I've added line notes to the lines you want to be changed. Thanks for your feedback and please let us know about other mistakes you may find.

**Comment by fredev** [comment]
Yes I am interesting too @MarcelGerber.

I found another error in the translation file 'locales/it/translation.json' on line 29:

in: `L'estensione Extract per Brackets (Anteprima) to permette`

change `to` with `ti`

**Comment by marcelgerber** [comment]
Thanks you two. Let's wait until @nicolo-ribaudo fixes those issues so I can then merge the changes.

**Comment by fredev** [comment]
@MarcelGerber Another little error in the contribute (contribuisci) page: the dots after 'Issues' and 'Features' are different, please check it.
schermata 2015-05-14 alle 12 01 20

**Comment by nicolo-ribaudo** [comment]
@MarcelGerber Fixed

**Comment by marcelgerber** [comment]
Merged. Thank you all so much!

**Pull request merged**